WinSW.nuspec

```diff
@@ -27,7 +27,9 @@ More info about the wrapper is available in the projects GitHub repository.
   <files>
     <file src="artifacts\WinSW.NET2.exe" target="lib\net20-full\WinSW.NET2.exe" />
     <file src="artifacts\WinSW.NET4.exe" target="lib\net40-full\WinSW.NET4.exe" />
+    <file src="artifacts\WinSW.NET461.exe" target="lib\net461-full\WinSW.NET461.exe" />
```

Comment:
Only .NET 4.0 has the concepts of "Client Profile" and "Full Profile". The problem with `net20-full` and `net40-full` is people may be using hard-coded paths. But we can omit the suffix for `net461`.

Reply:
Would be nice to update it before the release, will do it. Thanks!